### ISSUE TITLE
[0.9.6.0 - 0.9.704] Docking port does not transfer power directly

### ISSUE THREAD

**Issue by rav2n** [Regalis11/Barotrauma#2217]
- [x] I have searched the issue tracker to check if the issue has already been reported.

**Description**
If, say, the shuttle has batteries connected directly to the docking port, then they will not work. In fact, I do not think this is a problem, but someone may not like it, that you need to add another junction box or relay to power.

**Steps To Reproduce**
Connect any energy consumer directly to the docking port and try to power it from the other side. I will also attach a demo assembly with this problem.

**Version**
0.9.6.0 - 0.9.704

**Additional information**

[Test docking port transfers.zip](https://github.com/Regalis11/Barotrauma/files/4173758/Test.docking.port.transfers.zip)

**Comment by david-sharer** [comment]
I ran in to this while learning to make ships and it was pretty confusing. My bet is that it is a regression, as a shuttle I grabbed from elsewhere also had the problem. (Tombstone)

Additionally, it seems to account for all attached items when pulling power, but the power does not get provided to the items. So it shows as overvoltage.
![image](https://user-images.githubusercontent.com/26589071/86880687-e73ca380-c0a1-11ea-86f7-9c340118ff00.png)
![image](https://user-images.githubusercontent.com/26589071/86880739-fa4f7380-c0a1-11ea-88fb-57ab7efbfc3b.png)

Top battery + light is connected directly to the docking hatch.
Bottom battery + light is connected via relay.
Seems to be accounting for both of them in Power, but then the actual load is only half of them.
